package.json

```diff
@@ -23,6 +23,11 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "nodemon": "^1.9.1"
+    "bourbon": "^4.2.6",
+    "bourbon-neat": "^1.7.4",
```

Comment:
Those modules are being used for dev only, not needed once in production (eg. static server serves my sandbox, bourbon just installs a sass lib to compile into production css